prds/587-github-copilot-provider.md

```
@@ -0,0 +1,382 @@
+}
+
+export async function acquireCopilotToken(opts: {
+  clientId: string;       // GitHub OAuth App client ID registered for dot-ai
+  cachePath: string;      // File path for token persistence
+  logger: Logger;
+}): Promise<CopilotTokenResult>
+```
+
+**Flow:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)